panel/src/components/Views/Login/index.js

```diff
@@ -0,0 +1,28 @@
+import LoginView from "./LoginView.vue";
+
+import LoginForm from "./LoginForm.vue";
+import LoginAlert from "./LoginAlert.vue";
+import LoginCodeForm from "./LoginCodeForm.vue";
+
+import InstallationView from "./InstallationView.vue";
+import ResetPasswordView from "./ResetPasswordView.vue";
```

Comment:
I love that the we reorganize the login components like this. But I don't really think of the Installation View as a part of this to be honest. I'm not entirely sure about the ResetPasswordView. It's a weird hybrid somewhere between account and login.

Reply:
Moved InstallationView to its own folder and ResetPasswordView to users. It's a hybrid, but in the end maybe more account related as you need a user first with a password to be able to reset it.